### DISCUSSION TITLE
useFetch manipulate error response

### DISCUSSION THREAD

**Discussion by danillo-hilbs** [nuxt/nuxt#17259] [Questions] [answered]
Guys, does anyone know how I can access values ​​returned in an Error? I would like to expose in the return of this composable items such as: error.message, error.statusCode

I'm using useFetch()

![image](https://user-images.githubusercontent.com/34604307/179101324-5f503702-ee9d-478d-8260-8a6d151571df.png)

**Comment by mirumirumi** [comment, marked as answer]
Hi!
You can get the content of the error in a very simple way, how does what you are looking for differ from this?
If you want to retrieve the message body or status code after accessing the HTTP endpoint, you can do so by using conventional methods such as `axios`.
You then wrap it in your own `useFetchApi`.

(Please note that this can be null if no error has occurred)

```html
// app.vue

<template>
  <div>
  </div>
</template>

<script setup lang="ts">
const { data, error } = useFetch("/api/hi")

console.log(error)
</script>
```

```ts
// server/api/hi.ts
export default defineEventHandler(async (event) => {
  throw new Error("this is error.")
})
```

It shows in console:

```bash
[nuxt] [request error] this is error.
... (and more stack traces...)
```
